Makefile

```diff
@@ -141,21 +139,25 @@ endif
 
 ##@ Deployment
 
+ifndef ignore-not-found
+  ignore-not-found = false
+endif
+
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
 	## TODO: Remove sed usage after all v1alpha1 references are updated to v1beta1 in codebase.
 	## For local testing, conversion webhook defined in crd makes call to webhook for each v1alpha1 reference
 	## causing failures as we don't set up the webhook for local testing.
 	$(KUSTOMIZE) build config/crd | sed '/conversion:/,/- v1beta1/d' |kubectl apply --server-side=true -f -
 
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl delete -f -
+	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
```

Comment:
Why do we need this?

Reply:
It is mentioned in the upgrade docs - https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.16.0/#for-golang-based-projects-add-the-flag-ignore-not-found-into-the-makefile-for-the-uninstall-and-undeploy-targets